### PR TITLE
Calypso Colors: add Google Blue color

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request

* Add Google's Blue color in addition to the existing Social Media colors
* See https://github.com/Automattic/color-studio/pull/30

#### Testing instructions

* Not too sure how to test this within the Calypso repo.

### CODE REVIEW

File: packages/calypso-color-schemes/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/calypso-color-schemes",
-	"version": "2.1.1",
+	"version": "2.2.0",
```
Comment (porada) [inline packages/calypso-color-schemes/package.json:3]: Minor version bump should be enough for this change →  `2.1.2`.

File: packages/calypso-color-schemes/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 2.2.0
+
+* Introduce Google Color
```
Comment (porada) [inline packages/calypso-color-schemes/CHANGELOG.md:5]: Past release notes specifically mention added or removed properties. Suggested change:

```
## 2.1.2

* Added Google brand color:
    * `--color-google`
```

**Review by porada — CHANGES_REQUESTED** [review]

**Comment by porada** [comment]
In order to test this, you can set any element’s background to `var(--color-google)`. If the specific blue appears on the screen, that should be enough to confirm that the property has been included in the build.

File: packages/calypso-color-schemes/CHANGELOG.md (hunk 2 above)
Comment (jeherve) [reply packages/calypso-color-schemes/CHANGELOG.md]: Thank you for the feedback. Done in 5483ec93a467ee8f3675821386635dfe25af79ca

File: packages/calypso-color-schemes/package.json (hunk 1 above)
Comment (jeherve) [reply packages/calypso-color-schemes/package.json]: Done in 5483ec93a467ee8f3675821386635dfe25af79ca

**Comment by obenland** [comment]
@jeherve Is this still current?

**Comment by jeherve** [comment]
> Is this still current?

No, we ended up building the feature without any colors. Closing this now.

**Pull request closed without merging**